app/views/layout.hbs

```diff
@@ -72,11 +72,11 @@
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <script src="http://code.jquery.com/jquery-2.1.4.js"></script>
     <!--underscore-->
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.3.3/underscore-min.js" type="text/javascript"></script>
+    <script src="http://underscorejs.org/underscore.js" type="text/javascript"></script>
     <!--backbone-->
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/backbone.js/0.9.2/backbone-min.js" type="text/javascript"></script>
+    <script src="http://backbonejs.org/backbone.js" type="text/javascript"></script>
```

Comment:
You should consider hosting these locally, since we are making use of npm anyways.